bibliopixel/drivers/LPD8806.py

```diff
@@ -35,23 +32,23 @@ def __init__(self, num,
         "display": "LPD8806 (SPI Native)",
         "desc": "Interface with LPD8806 strips over a native SPI port (Pi, BeagleBone, etc.)",
         "params": [{
-                "id": "num",
-                "label": "# Pixels",
-                "type": "int",
-                "default": 1,
-                "min": 1,
-                "help": "Total pixels in display."
+            "id": "num",
```

Comment:
Interesting - your indentation seems to be more accurate than what's there!  In principle I'm against mixing whitespace changes with actual code changes but we're playing fast and loose here.  :-D